Chapters/1-4-Chapter4.md

```diff
@@ -0,0 +1,232 @@
+## Chapter 4
+
+The station was just filling up for the next train after the day's first nap. The warm and sweet aroma of freshly-baked handcakes caressed my nose and pulled me closer to Gilli's bakery at the far corner of the station. I figured that I'd earned something nice, so I turned to let my nose take me closer. But Laen had been waiting for his moment, and he bolted toward the bakery as soon as he saw me stop.
```

Comment:
> I figured that I'd earned something nice, so I turned to let my nose take me closer.
Hit even harder. What about "I figured this may be my last visit to Gilli's..."

Reply:
I don't know that Liffa would know/think this. A short stay in Ulukoma wouldn't mean much to hae. Gilli's would still be there after she came back.